### PR TITLE
Fix bug in letter of with numeric inputs

### PR DESCRIPTION
**Pull request by cycomachead**
Coerce input to string before checking length in LETTER OF

### CODE REVIEW

File: src/threads.js [hunk 1]
```diff
@@ -2706,14 +2706,14 @@ Process.prototype.reportLetter = function (idx, string) {
     if (string instanceof List) { // catch a common user error
         return '';
     }
+    str = isNil(string) ? '' : string.toString();
```
Comment (jmoenig) [inline src/threads.js]: we probably want to declare this as a local variable (```var str 0 ...```), so it doesn't accidentally become global...

**Comment by jmoenig** [comment]
Thanks for the quick fix, Michael!

File: src/threads.js (hunk 1 above)
Comment (cycomachead) [reply src/threads.js]: I think github cuts it off from the view, but the bar is rare in the existing definition. :)

Comment (jmoenig) [reply src/threads.js]: wow, you're right. That's funny!

**Pull request merged**